rundeckapp/grails-app/domain/rundeck/Execution.groovy

```diff
@@ -123,9 +124,15 @@ class Execution extends ExecutionContext {
                                                 (status in ['false', 'failed']) ? ExecutionService.EXECUTION_FAILED :
                                                         ExecutionService.EXECUTION_STATE_OTHER
     }
+
+    public boolean hasExecutionEnabled() {
+        return scheduledExecution.hasExecutionEnabled();
```

Comment:
the execution might not have a ScheduledExecution, so scheduledExecution might be null.

perhaps:

    return !scheduledExecution || scheduledExecution.hasExecutionEnabled()